src/components/product-tile/ProductTile.tsx

```diff
@@ -64,11 +70,24 @@ export const ProductTile: React.FC<{ product: Product }> = ({ product }) => {
         alt={product.name as string}
       />
       <div>
+        <button
+          style={{ marginRight: '10px' }}
+          onClick={() => changeLanguage('en')}
+        >
+          EN
+        </button>
+        <button
+          style={{ marginRight: '10px' }}
+          onClick={() => changeLanguage('pt')}
+        >
+          PT
+        </button>
+        <button onClick={() => changeLanguage('fn')}>FN</button>
         <h3 className="product-tile-title">{product.name}</h3>
         <h4 className="product-tile-price">£{product.price}</h4>
       </div>
       <Button handleClick={handleClick} iconName="shopping-bag">
-        Add to cart
+        {t('button')}
```

Comment:
Use an addToCart key instead. The keys should reflect the content of the message displayed rather than the element that displays them